source/GameInterface/Serialization/Impl/PerkObjectBinaryPackage.cs

```diff
@@ -25,7 +25,7 @@ public override void Pack()
 
         protected override void UnpackInternal()
         {
-            Object = MBObjectManager.Instance.GetObject<PerkObject>(StringId);
+            MBObjectManager.Instance?.GetObject<PerkObject>(StringId);
```

Comment:
Revert this change. Only getting the object will get rid of it when the function scope is left. Also remove the `?` as we want to be sure the object is retrieved